src/main/java/com/passbook/sparkeighteen/peristence/repository/TransactionRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.passbook.sparkeighteen.peristence.repository;
+
+import com.passbook.sparkeighteen.peristence.entity.TransactionEntity;
+import com.passbook.sparkeighteen.peristence.entity.UserEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface TransactionRepository extends JpaRepository<TransactionEntity, Integer> {
+    Optional<TransactionEntity> findById(BigInteger integer);
+
+    Optional<List<TransactionEntity>> findByUser(UserEntity user);
```

Comment:
You don't need to wrap a collection in Optional. Collection will be empty if there are no results found.

Reply:
Updated, As perviously discussed need this to be closed